internal/scheduler/scheduler.go

```diff
@@ -31,6 +31,12 @@ func (s *scheduler) Start(ctx context.Context, interval time.Duration) {
 		} else {
 			s.logger.Info().Msg("re-scheduling all completed syncs to run again")
 		}
+
+		if err := s.db.CleanQueueLogs(ctx, 30); err != nil {
+			s.logger.Err(err).Msg("encountered error cleaning queue logs")
+		} else {
+			s.logger.Info().Msgf("successfully removed repo sync jobs older than %v days", 30)
```

Comment:
```suggestion
			s.logger.Info().Msgf("successfully removed repo sync jobs older than %d days", 30)
```

just wondering - does it have to be `%d` since it's an int?

Reply:
good question

Reply:
Seems like it does , is not breaking at building it